test/support/ssl/insecure-cert.pem

```diff
@@ -0,0 +1,34 @@
+-----BEGIN CERTIFICATE-----
+MIIF9DCCA9ygAwIBAgIJAJhLP5JrvONkMA0GCSqGSIb3DQEBBQUAMFkxCzAJBgNV
```

Comment:
Yes, https server requires a working certificate.
With davewasmer/devcert, OS will prompt for elevated permissions to install a root CA. That's pretty invasive and I'm not sure if CI can accept it.
This listens on a random port, only for the duration of the test, and only exposes the test fixture directory to reads--curious to see how that could be a security threat. What was the webpack-dev-server problem? I don't see an [issue listed](https://github.com/webpack/webpack-dev-server/issues?utf8=%E2%9C%93&q=is%3Aissue%20%20certificate).